pocs/camera/camera.py

```diff
@@ -311,6 +311,22 @@ def set_property(self, prop, val):
         # Forces the command to wait
         self.get_command_result()
 
+    def set_property_values(self, prop_vals):
+        """ Sets a number of properties all at once
+
+        Args:
+            prop_vals (dict): A dict with keys corresponding to the property to
+            be set and values corresponding to the index option
+        """
+        set_cmd = list()
+        for prop, val in prop_vals.items():
+            set_cmd.append('--set-config-index {}={}'.format(prop, val))
```

Comment:
Is there a need to escape either the prop or val, or to otherwise enclose in quotes?

Reply:
This method is only setting the index and worked with minimal testing, but the property here can be a `/` delimited string so might make sense. I'll test out more when I get back to lab later today.

Reply:
I realize that I haven't checked whether shell=True is being passed into the subprocess, which determines whether that array is concatenated into a single string, or is treated as a raw argv.